### DISCUSSION TITLE
Where to get a lot of SWFs for testing?

### DISCUSSION THREAD

**Discussion by eiglow** [lightspark/lightspark#872] [Q&A] [answered]
Where are good places to get a lot of swf files for testing? I used to have a lot, but I lost them a few months ago, so I have just been going back to old issues and downloading swfs from there. This is pretty inconvenient though, is there a quicker way to get a lot of swfs for testing?

**Comment by huwdp** [comment, marked as answer]
I download .swf files from the resources below.

https://archive.org/download/armorgames
https://archive.org/download/swfusable/SWFS/

I think there is also a 874GB collection here (https://bluemaxima.org/flashpoint/downloads/) https://bluemaxima.org/flashpoint/datahub/Game_Master_List#Flash.
